packages/utils/src/string.ts

```diff
@@ -112,42 +120,49 @@ export function isMatchingPattern(value: string, pattern: RegExp | string): bool
  * @returns A base64-encoded version of the string
  */
 export function unicodeToBase64(plaintext: string): string {
-  const global = getGlobalObject();
-
-  // Cast to a string just in case we're given something else
-  const stringifiedInput = String(plaintext);
-  const errMsg = `Unable to convert to base64: ${
-    stringifiedInput.length > 256 ? `${stringifiedInput.slice(0, 256)}...` : stringifiedInput
-  }`;
+  const globalObject = getGlobalObject() as GlobalWithBase64Helpers;
 
   // To account for the fact that different platforms use different character encodings natively, our `tracestate`
   // spec calls for all jsonified data to be encoded in UTF-8 bytes before being passed to the base64 encoder.
   try {
+    if (typeof plaintext !== 'string') {
+      throw new Error(`Input must be a string. Received input of type ${typeof plaintext}.`);
```

Comment:
```suggestion
      throw new Error(`Input must be a string. Received input of type ${Object.prototype.toString.call(plaintext)}.`);
```

This will be way more descriptive.

Reply:
Actually, the user already gets the stringified version, on the line immediately above:

```
Uncaught SentryError: Unable to convert to base64: Symbol(thing).
Got error: Error: Input must be a string. Received input of type 'symbol'.

Uncaught SentryError: Unable to convert to base64: 4.
Got error: Error: Input must be a string. Received input of type 'number'.

Uncaught SentryError: Unable to convert from base64: [object Object].
Got error: Error: Input must be a string. Received input of type 'object'.

Uncaught SentryError: Unable to convert to base64: [object TextDecoder]
Got error: Error: Input must be a string. Received input of type 'object'.

// etc.
```

I did switch it to `JSON.Stringify()`, because I tested it out, and there are more kinds of inputs that come out the right way with that.